dbms/src/AggregateFunctions/AggregateFunctionNull.h

```diff
@@ -91,13 +90,23 @@ class AggregateFunctionNullBase : public IAggregateFunctionHelper<Derived>
     }
 
 public:
-    explicit AggregateFunctionNullBase(AggregateFunctionPtr nested_function_)
+    explicit AggregateFunctionNullBase(AggregateFunctionPtr nested_function_, bool need_counter = false)
         : nested_function{nested_function_}
     {
         if (result_is_nullable)
             prefix_size = nested_function->alignOfData();
         else
             prefix_size = 0;
+
+        if (result_is_nullable && need_counter)
+        {
+            auto tmp = prefix_size;
+            prefix_size += sizeof(Int64);
+            if ((prefix_size % tmp) != 0)
+                prefix_size += tmp - (prefix_size % tmp);
```

Comment:
I think the final `prefix_size` should be divided by `alignOfData`. But this code doesn't look it's guaranted.